cdap-e2e-tests/src/e2e-test/features/namespaceadmin/NameSpaceAdmin.feature

```diff
@@ -0,0 +1,21 @@
+@Namespaceadmin
+Feature: NameSpaceAdmin - Validate system admin page flow
+
+  @Namespaceadmin
+  Scenario:Validate user is able to create new namespace preferences inside namespace admin
```

Comment:
How are we validating the preferences we set ?

Reply:
Here we are adding only the design time scenarios we will be adding the validation steps when the requirement for creation with end to end tests is planned . The scope was to create sanity test suite to start with these features . We are in connect with google team to define the future scope which will be added as end to end scenarios in this month . 
As of now the plan was to create the feature and to check if the tests are passing when the integration is done with e2e FW .